### DISCUSSION TITLE
Don't timeout codespace when typing in terminal

### DISCUSSION THREAD

**Discussion by icecream17** [community/community#52210] [Codespaces]
### Select Topic Area

Product Feedback

### Body

I assume the codespace doesn't expire (timeout) when editing a file, so could this happen for when typing in a terminal? (Can't rely on any stdout because typing can be sent to a program or something, probably the only way is html Event detection)

It's really annoying to lose all my progress when the codespace just ~~randomly~~ decides to stop

it's 10:27 utc, so https://github.com/orgs/community/discussions/41522 doesn't apply

**Comment by aishraj** [comment]
Hello there! We would suggest you to take a look at the[ timeout settings](https://docs.github.com/en/codespaces/customizing-your-codespace/setting-your-timeout-period-for-github-codespaces) for your codespace. Additionally, we would appreciate more information on the client that you are using so that we can have someone from our end look into this.

**Reply by icecream17** [reply]
I opened codespaces using the github website.

Incidentally, is there any way to change the timeout setting for a particular repository?

More information, <s>but probably irrelevant since</s> the timeout seems to exactly match what's happening:
`navigator.userAgent` is `Mozilla/5.0 (X11; CrOS x86_64 14541.0.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36`; whatismybrowser `Looks like Chrome on Linux but announcing (Chrome 108 userAgent)`; chrome://version: `108.0.5359.172`; Lenovo 300e Chromebook 2nd Gen AST
